File: flutter/lib/models/karyawan_model.dart
class Karyawan {
  int? id;
  String? nama;
  int? idJabatan;
  String? jabatan;
  String? gender;
  int? usia;
  String? menikah;
  int? jlhAnak;
  String? alamat;
  String? noTelp;
  String? noRek;
  String? email;
  String? username;
  int? createdBy;
  int? updatedBy;

  Karyawan.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    nama = json['nama'];
    idJabatan = json['id_jabatan'];
    jabatan = json['jabatan'];
    gender = json['gender'];
    usia = json['usia'];
    menikah = json['menikah'];
    jlhAnak = json['jlh_anak'];
    alamat = json['alamat'];
    noTelp = json['no_telp'];
    noRek = json['no_rek'];
    email = json['email'];
    username = json['username'];
    createdBy = json['created_by'];
    updatedBy = json['updated_by'];
  }
}

File: api/src/models/models/karyawanModel.js
module.exports = (sequelize, DataTypes) => {
  const Karyawan = sequelize.define('Karyawan', 
  {
    id: {
      type: DataTypes.INTEGER,
      autoIncrement: true,
      primaryKey: true
    },
    nama: {
      type: DataTypes.STRING,
      allowNull: false
    },
    gender: {
      type: DataTypes.STRING,
      allowNull: false
    },
    usia: {
      type: DataTypes.INTEGER,
      allowNull: false
    },
    menikah: {
      type: DataTypes.STRING,
      allowNull: false,
      defaultValue: 'BELUM'
    },
    jlh_anak: {
      type: DataTypes.INTEGER,
      allowNull: false,
      defaultValue: 0
    },
    alamat: {
      type: DataTypes.STRING,
      allowNull: false
    },
    no_telp: {
      type: DataTypes.STRING,
      allowNull: false
    },
    no_rek: {
      type: DataTypes.STRING,
      allowNull: false
    },
    status: {
      type: DataTypes.STRING,
      allowNull: false,
      defaultValue: 'ACTIVE'
    }
  }, {
    tableName: 'karyawan',
    createdAt: 'created_at',
    updatedAt: 'updated_at'
  })

  return Karyawan
}